apps/src/util/experiments.js

```diff
@@ -22,7 +22,7 @@ experiments.REDUX_LOGGING = 'reduxLogging';
 experiments.SCHOOL_AUTOCOMPLETE_DROPDOWN_NEW_SEARCH =
   'schoolAutocompleteDropdownNewSearch';
 experiments.APPLAB_DATASETS = 'applabDatasets';
-experiments.ASSIGNMENT_UPDATES = 'assignmentUpdates';
+experiments.CENSUS_MAP_ON_MAPBOX = 'censusMapOnMapbox';
```

Comment:
Is this supposed to be here?

Reply:
whoops - nope! leftover from an earlier merge conflict, but was removed from staging in #31903. Good catch!